build.sbt

```diff
@@ -33,3 +33,4 @@ lazy val `sbt-api-builder` = project
     scriptedBufferLog := false,
     scriptedLaunchOpts ++= Seq("-Xmx1024M", s"-Dplugin.version=${version.value}")
   )
+  .enablePlugins(SbtPlugin)
```

Comment:
This replaces most of the `scriptedX` defaults (not the two above AFAICT) _and_ also the need for `sbtPlugin := true`